src/components/Header/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { Grid } from '@material-ui/core'
+import Presentationbar from '../Presentationbar/index'
+import { connect } from 'react-redux'
+import {
+        toggleModal, 
+        setActivePresentation,
+        setPresentationTitle,
+        createPresentation
+        }  from '../../actions/index'
+
+
+const Header = ({
+                    toggleModal, 
+                    isModalActive, 
+                    presentations, 
+                    activePresentation,
+                    setPresentationTitle,
+                    createPresentation,
+                    setActivePresentation
+                }) => {
+    return (
+        <Grid container>
+            <Grid item xs={4}>
+                <Presentationbar 
+                    toggleModal={toggleModal} 
+                    isModalActive={isModalActive}
+                    presentations={presentations} 
+                    activePresentation={activePresentation}
+                    setPresentationTitle={setPresentationTitle}
+                    createPresentation={createPresentation}
+                    setActivePresentation={setActivePresentation}
+                />
+            </Grid>
+        </Grid>
+    )
+}
+
+const mapStateToProps = state => {
+    return {
+        slides : state.presentation.get('slides').toJS(),
+        activePresentation: state.presentation.get('active_presentation'),
+        presentations: state.presentation.get('presentations').toJS(),
```

Comment:
We should remove the .toJS() call on this, as it could incur a performance hit

Reply:
Removed component